externs/shaka/player.js

```diff
@@ -1400,6 +1400,7 @@ shaka.extern.StreamingConfiguration;
  * @typedef {{
  *   codecSwitchingStrategy: shaka.config.CodecSwitchingStrategy,
  *   addExtraFeaturesToSourceBuffer: function(string): string,
+ *   sourceBufferExtraFeatures: string,
```

Comment:
I think this is a great candidate to handle the way we normally do.  See `'forceTransmuxTS'` handling where we ingest configs in `lib/player.js`.